### ISSUE TITLE
The application policy for destroy isn't working properly

### ISSUE THREAD

**Issue by UJKhokhar** [UJKhokhar/blocipedia#5]
Logged in as a user who created the Wiki, I noticed the issue when the destroy button wasn't appearing in the ```wikis/show.html.erb``` file. Specifically this code: 
```
<% if policy(@wiki).destroy? %>
  <%= link_to 'Delete Wiki', @wiki, class: 'btn btn-danger', 
          method: :delete, 
          data: { confirm: 'Are you sure you want to delete your Wiki'} %>
<% end %>
```

Here is the destroy policy in ```policies/application_policy.rb```
```
def destroy?
  record_owned_by_user? || user_is?('admin')
end

def record_owned_by_user?
  return false if record.user.nil?
  return false unless user_exists?
  record.user == user
end

def user_exists?
  user.present?
end

```

**Comment by UJKhokhar** [comment]
After inspecting ```policies/application_policy.rb``` I edited the file so that:
```
def destroy?
  record_owned_by_user?
end
```
This still returns false for my user for some unknown reason.

**Comment by UJKhokhar** [comment]
In an attempt to further simplify the issue:
```
def destroy?
  record.user == user
end
```
This also returned false. So the issue lies not with the application policy, but with the Wiki's not being associated with users.

**Comment by UJKhokhar** [comment]
After inspecting an instance of Wiki, I saw that ```user_id: nil```

I fixed the issue with this commit: b481167
